express-app/controllers/question.js

```diff
@@ -18,8 +18,8 @@ module.exports = {
       descripcion: pregunta.descripcion,
       orden: pregunta.orden,
     });
-    pregunta.opcionesPregunta.forEach(function (item) {
-      question.opcionesPregunta.push(AnswerOption.addAnswerOption(item));
+    pregunta.opciones_pregunta.forEach(function (item) {
```

Comment:
I can't let this in camel case because is the name that I retrieved from API. If I put in camel case creates an error.